platform/windows/joypad_windows.cpp

```diff
@@ -139,6 +143,7 @@ bool JoypadWindows::is_xinput_device(const GUID *p_guid) {
 }
 
 bool JoypadWindows::setup_dinput_joypad(const DIDEVICEINSTANCE *instance) {
+	ERR_FAIL_NULL_V_MSG(dinput, false, "DirectInput not initalized. Rebooting your PC may solve this issue.");
```

Comment:
Typo "init**i**alized" (same in other error).

Reply:
Fixed.